src/plugins/controls/public/control_types/range_slider/range_slider_embeddable.tsx

```diff
@@ -307,12 +327,35 @@ export class RangeSliderEmbeddable extends Embeddable<RangeSliderEmbeddableInput
     rangeFilter.meta.type = 'range';
     rangeFilter.meta.params = params;
 
-    this.updateComponentState({ loading: false });
+    // Check if new range filter results in no data
+    if (!ignoreParentSettings?.ignoreValidations) {
```

Comment:
That's fair. In that case, we should add checks for ignoring filters, query, and time range within this block individually. 

I just wasn't sure if we wanted to apply the query sync settings here because the goal here is query ES with the newly built range filter and check if the panels end up with no data. If all of these global filters/queries were ignored in this check, we could end up with a "valid" selection but still have no data in the panels.

Reply:
Good point! :)  If we have no plans to turn these settings back on, probably not be worth adding all that - but might be better to make sure range slider supports the same features as options list? 🤷 Maybe @ThomThomson has some input on whether or not this is worth it!

Reply:
Just to formalize what we discussed in the sync earlier, the choice is to ignore adding this for now, and if we ever re-introduce the Ignore Settings we'll dig into this then and just treat it as a bug.